### PR TITLE
Remove all mythx mentions

### PR DESCRIPTION
**Pull request by Nurchik**

### CODE REVIEW

**Comment by Nurchik** [comment]
@joaosantos15 could you please review this PR?

**Comment by Nurchik** [comment]
@dmuhs could you please review this PR?

File: AUTHORS.rst [hunk 1]
```diff
@@ -11,4 +11,4 @@ Development Lead
 Contributors
 ------------
 
-None yet. Why not be the first?
+* Nuradil Alymkulov <nurchik17@gmail.com>
```
Comment (JoranHonig) [inline AUTHORS.rst:14]: Nice 😂

**Comment by joaosantos15** [comment]
I tested it and everything seems to be working. MythX mentions were removed, `.fuzz.yml` is now the default config file and the error messages didn't show stack traces, but I only tested a few error cases.

**Review by joaosantos15 — APPROVED** [review]
Look good to me, I also checked the makefile commands and they were working. Let's merge and make a new minor version release.

File: AUTHORS.rst (hunk 1 above)
Comment (Nurchik) [reply AUTHORS.rst:14]: @JoranHonig could I leave it here?

**Pull request merged**